rocketpy/prints/sensors_prints.py

```diff
@@ -74,7 +74,7 @@ def orientation(self):
         self._print_aligned("Orientation:", self.sensor.orientation)
         self._print_aligned("Normal Vector:", self.sensor.normal_vector)
         print("Rotation Matrix:")
-        for row in self.sensor.rotation_matrix:
+        for row in self.sensor.rotation_sensor_to_body:
```

Comment:
maybe you should emphasize in the print message that you're using the `rotation_sensor_to_body` matrix rather than the other option. What do you think?